<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DynamicRules</class>
 <widget class="QWidget" name="DynamicRules">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="LineEdit" name="nameText" native="true">
     <property name="placeholderText" stdset="0">
      <string>Name of Dynamic Rules</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="4">
    <widget class="ListView" name="rulesList"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="addBtn">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="editBtn">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="removeBtn">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>53</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="UrlLabel" name="aboutLabel">
     <property name="text">
      <string>About Rules</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UrlLabel</class>
   <extends>QLabel</extends>
   <header>urllabel.h</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QWidget</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>ListView</class>
   <extends>QListView</extends>
   <header>listview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
